pages/your_domain/human_pathogen_genomics.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Human pathogen genomics
+search_exclude: true
+description: <!---REPLACE THIS with a one sentence description of the page--->
```

Comment:
```suggestion
```

hackmd does not allow me as editors to manipulate the content of the PR on GitHub so I do it through a suggestion :)

Reply:
Not sure how to open the menu to change the setting and the [instructions from GitHub seem to be incomplete](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork). Maybe the setting is not available since the repository we’ve been working in is owned by an organisation and not an individual?